### ISSUE TITLE
[BUG] Starting Mapepire server produces Object not found error

### ISSUE THREAD

**Issue by AndyYouens** [ThePrez/ServiceCommander-IBMi#230]
**Describe the bug**
Starting the mapepire server produces a language object not found.

**To Reproduce**
This can be seen in the server's joblog:
CALL PGM(QP2SHELL2) PARM('/QOpenSys/pkgs/bin/bash' '-c' 'cd /QOpenSys/pkgs
/lib/mapepire && exec ../../bin/mapepire >> /HOME/ANDY/.sc/logs/2024-09-24
-17.56.31.mapepire.log 2>&1')                                             
**Object not found.  Object is EN_US.UTF-8.**                                 
Job 393313/ANDY/MAPEPIRE changed by ANDY.                                 

SC was used to start the server from a SSH session.
No errors seen when using SC to start the server job.

This is happening on OS versions 7.3 7.4 & 7.5

**Comment by chrjorgensen** [comment]
@AndyYouens I just did a quick test on my sandbox running IBM i 7.5: `yum install mapepire-server` followed by `sc start mapepire` - and no problems found.

Could it be some locale missing on the system? 

EDIT:
> Can you run other YUM utilities in with locale EN_US in UTF-8, e.g. `LC_ALL=EN_US.UTF-8 /QOpenSys/pkgs/bin/mc`?

Bad suggestion - it runs even when specifying a non-existing locale...

The command `locale -a` will show all locales found by PASE. Is `EN_US.UTF-8` in the list output by the command?

**Comment by AndyYouens** [comment]
I dont think it has anything to do with locales being missing,  I;ve tried on three sep boxes in the uk all with the same results.

no problems overall with sc

EN_US.UTF-8 is there.

**Comment by AndyYouens** [comment]
@chrjorgensen after starting mapepire did you check its joblog?

**Comment by chrjorgensen** [comment]
@AndyYouens Ah, now I see what you mean - sorry, missed you're talking about the server joblog!

Yes, I have the same error message in the server joblog on my 7.5 system... :disappointed: 

Does it have any impact on the operation of the mapepire server?

**Comment by AndyYouens** [comment]
Yes, I cannot get any connection or data from the mapepire server.  My python client is taken from the sample example [here.](https://mapepire-ibmi.github.io/guides/usage/python/)

**Comment by richardschoen** [comment]
Andy. Not sure if this helps, but this is what I do:
 ```
bash /QOpenSys/pkgs/bin/mapepire
```

Maybe try launching it from an SSH session to see if you get same issue.

**Comment by richardschoen** [comment]
> Yes, I cannot get any connection or data from the mapepire server. My python client is taken from the sample example [here.](https://mapepire-ibmi.github.io/guides/usage/python/)

Did you select ignoreunauthorized=True ?

**Comment by AndyYouens** [comment]
now getting data when starting mapepire directly.  no errors seen in joblogs. Proves problem is within SC.
